src/routes/api/contest.route.js

```diff
@@ -1,7 +1,8 @@
 const express = require('express');
 const router = express.Router()
 const contestController = require('../../controllers/contest.controller')
+const auth = require('../../middlewares/authorization')
 
-router.get('/status', contestController.status);
+router.get('/status', auth, contestController.status);
```

Comment:
This makes me believe this endpoint is a health check of some sort, but upon investigation, the function is something else. I suggest renaming it to something more appropriate that follows a naming convention

Reply:
GET/v0/contests
GET/v0/contests/:contestId
GET/v0/rankings

Reply:
These status endpoints for **contests** and **users**, are placeholder for the time being. 
If you hit **GET/v0/contests/status** and **GET/v0/users/status**.

I have already configured the API routes to that naming convention in this PR: https://github.com/uosoftwareclub/core-service/pull/1